### PR TITLE
Malin's movie project

### PR DESCRIPTION
**Pull request by malin798**
Link to my deployed project:

https://malins-movie-project.netlify.com

My reflections on how this week's project turned out:

Things I'd like to have clarified or explained in more detail:

### CODE REVIEW

**Comment by malin798** [comment]
> Hi! The link or project doesn't work. The page is empty for me. Can you check on that?

Not quite sure what the problem is.. it works fine on chrome, firefox and edge for me... nothing shows up for you? Is there any errors so I can understand the problem? For me (and on my boyfriends computer) everything shows up as it's supposed to.

File: code/src/pages/Movie.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        </Link>
+      
+      <div className="selected-movie-details" key={movie.id}>
+        <img src={`https://image.tmdb.org/t/p/w342${movie.poster_path}`} alt="movie poster"></img>
+
+        <section className="movie-overview">
+          <h1>{movie.title}, <span className="rating">{movie.vote_average} /10 </span> </h1>
+          <p className="movie-summary">{movie.overview} </p>
+          <p className="genre"> Genre: &#160; {movie.genres.map((genre, index) => { 
+           console.log("genre.length", movie.genres.length)
```
Comment (ericaglimsholt) [inline code/src/pages/Movie.js:44]:
```suggestion
```

File: code/src/pages/Movie.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+           console.log("index", index)
```
Comment (ericaglimsholt) [inline code/src/pages/Movie.js:45]:
```suggestion
```

**Comment by ericaglimsholt** [comment]
It works now! Good work :)

**Pull request closed without merging**